zk-sdk/Cargo.toml

```diff
@@ -13,11 +13,13 @@ edition = { workspace = true }
 base64 = { workspace = true }
 bytemuck = { workspace = true }
 bytemuck_derive = { workspace = true }
+js-sys = { workspace = true }
 merlin = { workspace = true }
 num-derive = { workspace = true }
 num-traits = { workspace = true }
 solana-program = { workspace = true }
 thiserror = { workspace = true }
+wasm-bindgen = { workspace = true }
```

Comment:
What do you think about only requiring the wasm dependencies if the target is wasm, similar to what we're doing at https://github.com/anza-xyz/agave/blob/83e7d84bcc4cf438905d07279bc07e012a49afd9/sdk/pubkey/Cargo.toml#L40-L43

Reply:
Ah yes, I should have done this initially. Thanks!